src/components/cardbuilder/card.css

```diff
@@ -627,7 +627,7 @@ button {
 
 @media (min-width: 43.75em) {
     .overflowSquareCard, .overflowPortraitCard {
-        width: 23.3vw;
+        width: 23.1vw;
```

Comment:
These are all magic numbers, and they were changed to fit within the padding better.